### PR TITLE
Avoid infinite loop in EOF connections

### PR DESCRIPTION
**Pull request by pchico83**
Signed-off-by: Pablo Chico de Guzman <pchico83@gmail.com>

Users have reported infinite loops in this code if the error is `EOF`

### CODE REVIEW

File: pkg/ssh/forward.go [hunk 1]
```diff
@@ -77,6 +79,7 @@ func (f *forward) start(ctx context.Context) {
 			}
 
 			log.Infof("%s -> failed to accept connection: %v", f.String(), err)
+			<-tick.C
```
Comment (rberrelleza) [inline pkg/ssh/forward.go:82]: If we've only seen this on the reverse, I think we should only do it there for now.

**Review by rberrelleza — APPROVED** [review]
For future context: the infinite loop is by design, since we don't have a way of knowing from inside the container is there is a process to reverse forward to, we simply keep trying. This is consuming a lot of CPU, thus the ticket to wait a little bit in between retries.

File: pkg/ssh/forward.go (hunk 1 above)
Comment (pchico83) [reply pkg/ssh/forward.go:82]: Loops without a delay scare me

Comment (rberrelleza) [reply pkg/ssh/forward.go:82]: Ok

**Pull request merged**